### ISSUE TITLE
Source links in math mode

### ISSUE THREAD

**Issue by ktraunmueller** [ktraunmueller/Compositor#21]
- record all calls to `new_noad()` (§1136 ff)
- make sure to copy over source info when noad fields get copied (e.g., the nucleus in §1186)
- check all `mlist_to_hlist()` (§719 ff) helpers: `make_over()`, `make_under()`, `make_vcenter()`, `make_radical()`, `make_math_accent()`, `make_fraction()`, `make_op()`, `make_ord()`, `make_scripts()`, `make_left_right()`
- make sure nested macros work, e.g. `\hat{f(\omega)}` (depends on #26)

**Comment by ktraunmueller** [comment]
I will split this up into individual tickets for all math commands. Leaving it open for now as this is work in progress.

**Comment by ktraunmueller** [comment]
Created tickets for known issues, closing this.
